### ISSUE TITLE
task documentation outdated for custom tasks

### ISSUE THREAD

**Issue by martinjaggi** [epfml/disco#549]
in `docs/TASK.md`, parts of the custom task creation is already outdated:

such as `make sure to include its reference in the discojs/discojs-core/src/tasks/index.ts file as follows:`
it seems to me nothing has to be changed anymore in that file, right?

would be good if someone quickly goes through if things work to add a new task like described there

**Comment by martinjaggi** [comment]
i guess now it's a bit confusing if such a new tasks should become part of `https://github.com/epfml/disco/blob/develop/discojs/discojs-core/src/default_tasks/index.ts` or not, if one just wants to quickly hack and get a new task running?

**Comment by morganridel** [comment]
Well you can still add new tasks by creating your own `.ts` file and adding it to the `index.ts` file, but you need to modify the code/fork the project for that. these are the "default_tasks" and they are intended to be task provided by us to everyone using the library.

But now you can also add the task dynamically at runtime just before starting the server, which would be the preferred and "public" way  because you can do this easily by installing the server package from npm

Maybe we can stop encouraging people to adding their own task file directly in the code for their usage. Adding tasks directly in the code would be for us or people wanting to contribute on GitHub to the default tasks. WDYT @s314cy ?
